experiment/posttest.json

```diff
@@ -0,0 +1 @@
+[{"question": "1. We apply the experiment in close end condition.", "answers": {"a": "True", "b": "False"}, "correctAnswer": "b"}, {"question": "2. Longitudinal stress is equal to half the hoop stress in same pressure.", "answers": {"a": "True", "b": "False"}, "correctAnswer": "b"}, {"question": "3. In an open end condition, longitudinal stress is equal to zero.", "answers": {"a": "True", "b": "False"}, "correctAnswer": "b"}]
```

Comment:
I assume that we are waiting for the SME for the questions and the is the reason the questions are not following the new format

Reply:
Only the Simulation Folder part of the code was done. This requires help from SME.